tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/hive/HiveTableManager.java

```diff
@@ -96,7 +100,12 @@ public HiveTableManager(
         this.hiveThriftClient = hiveThriftClient;
         this.databaseName = databaseName;
         this.queryExecutor = checkNotNull(queryExecutor, "queryExecutor is null");
-        this.hdfsDataSourceWriter = checkNotNull(hdfsDataSourceWriter, "hdfsDataSourceWriter is null");
+        this.fsDataSourceWriter = checkNotNull(fsDataSourceWriter, "fsDataSourceWriter is null");
+        checkNotNull(testDataBaseFSPrefix, "testDataBaseFSPrefix is null");
+        if (testDataBaseFSPrefix.contains("***empty***")) {
```

Comment:
please remove this `if` with magic value comparison, why not pass empty value in configuration instead?